src/collections/domain/models/Collection.ts

```diff
@@ -0,0 +1,7 @@
+// NOTE: Props referenced from JsonPrinter.java
+export interface Collection {
+  id: number
+  name: string
+  alias: string
+  affiliation: string
+}
```

Comment:
Can we add `description` property to the Collection model as mentioned in the PR description?

Reply:
Added!